### DISCUSSION TITLE
A PR that used to run gitstream no longer runs gitstream

### DISCUSSION THREAD

**Discussion by michaelwiles** [linear-b/gitstream#430] [Troubleshooting]
A PR that has been running gitstream no longer runs gitstream and I have no idea why. It has been running successfully up till now but suddenly it just stopped.

Nothing relevant has changed.

And other PR's do still work (they do run gitstream).

![image](https://github.com/linear-b/gitstream/assets/1850356/0b3e492f-26df-41dc-ae49-f5d2a47f20f9)

I now get "gitstream workflow file not found".

We've put the gistream.cm file in .cm folder and the gitstream.yml file in /.github/workflows. Like I said, this has been working.

The gitstream action should run but it doesn't.

If you can point me to how I can debug this, that would be appreciated... However, it appears that it's not even running anything gitstream related.

The cm file has nothing special in it and it has not changed. It's just the default file from gitstream.

**Comment by vim-zz** [comment]
@michaelwiles what happen if you open a new PR in that same repo? does gitStream run successfully?

**Reply by michaelwiles** [reply]
That does work:

![image](https://github.com/linear-b/gitstream/assets/1850356/3d519002-61ce-408e-8614-be21dd1aacd8)
